wagtailinventory/tests/test_wagtail_hooks.py

```diff
@@ -125,7 +125,7 @@ def test_menu_item(self):
         self.assertFalse(item.is_shown(insufficient_permissions_request))
 
     def test_register_permissions(self):
-        self.assertQuerysetEqual(
+        self.assertQuerySetEqual(
```

Comment:
This error was seen once tests are run against Django 5.1.
Using this method name seems to still work OK with older versions of Django